msi/GitProduct.wxs

```diff
@@ -58,11 +62,25 @@
     <SetProperty Id="ENABLEFSCACHE" Value="[CMDLINE_ENABLEFSCACHE]" After="AppSearch" Sequence="first">CMDLINE_ENABLEFSCACHE</SetProperty>
 
     <!-- ENABLEGCM Property implementing the "Remember Property" pattern -->
-    <Property Id="ENABLEGCM" Value="1">
+    <!-- Initially set this to disabled until we can determine that .NET Framework 4.5.1 and above is installed. -->
+    <Property Id="ENABLEGCM" Value="0" />
```

Comment:
OK, so what you have _almost_ works, but it has to be tweaked for WiX. The good news is I started with what you have, modified it to fit the MSI installation process, and in the end, wound up with something much simpler. Thanks for making me think about this some more.